components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -174,13 +216,15 @@ private void handleClientError(ActionInvocationResponse.Builder builder, HttpEnt
             if (errorResponse != null) {
                 builder.response(errorResponse);
             } else {
-                builder.errorLog("Failed to execute the action request. Received: " + statusCode);
+                builder.errorLog("Failed to execute the action request for status code " + statusCode
+                        + " due to no error response is available.");
```

Comment:
This message is also wrong. The idea is an error is received with a particular status code

Reply:
Updated in